lib/models/subscription.js

```diff
@@ -109,6 +109,16 @@ module.exports = class Subscription extends Sequelize.Model {
     return queues.installation.add({ installationId, jiraHost })
   }
 
+  /*
+   * Returns array with sync status counts. [ { syncStatus: 'COMPLETED', count: 123 }, ...]
+   */
+  static async syncStatusCounts () {
+    const syncStatusCountQuery = 'SELECT "syncStatus", COUNT(*) FROM "Subscriptions" GROUP BY "syncStatus"'
+    const [ results ] = await this.sequelize.query(syncStatusCountQuery)
```

Comment:
Hrm, would it be better to do something like this?

```suggestion
    return this.sequelize.query(syncStatusCountQuery)
```

The only consumer right now uses it as a promise with the `then()` syntax (which is great!) but I don't think we need to `await` here do we?

Reply:
There's some subtly here.

`sequelize.query` returns an array containing two elements. We only want the first one. In order to only return that value, we need to await the query, extract the first element and return it.

I wasn't sure how to make it more obvious that there was multiple items in the array. When I made it explicit, the style checker yelled at me. 😄 

![image](https://user-images.githubusercontent.com/300976/65469451-58b1ab00-de1c-11e9-980f-86f8e9513488.png)

If you have a more obvious syntax, I'm eager for it!

Reply:
Ahh I see now. All good then!